lib/charms/mimir_coordinator_k8s/v0/mimir_cluster.py

```diff
@@ -418,11 +440,25 @@ def get_mimir_config(self) -> Dict[str, Any]:
                 coordinator_databag = MimirClusterProviderAppData.load(databag)
                 data = coordinator_databag.mimir_config
             except DataValidationError as e:
-                log.error(f"invalid databag contents: {e}")
+                log.info(f"invalid databag contents: {e}")
                 return {}
         return data
 
+    def get_loki_endpoints(self) -> Dict[str, str]:
```

Comment:
I'd consolidate this and the method above into one:

```
def _get_data_from_coordinator(self) -> Optional[MimirClusterProviderAppData]
def get_loki_endpoints(self):
    data = self._get_data_from_coordinator()
    if data: return data.loki_endpoints
    
def get_mimir_config(self):
    data = self._get_data_from_coordinator()
    if data: return data.mimir_config

```

Reply:
to DRY all those try/catch and guards

Reply:
Good point!